### PR TITLE
Use range instead of seperate templates for licensify

### PR DESCRIPTION
**Pull request by theseanything**
Each of the licensify application have a common set of templates that don't differ. Switching to using a range loop to create the resources for all apps from a single template, instead of duplicating the templates per app. Makes it much easier to make changes to common configuration.

### CODE REVIEW

File: charts/licensify/values.yaml [hunk 1]
```diff
@@ -18,43 +18,49 @@ clamav:
...
+    replicaCount: 1
+    port: 9950
+    healthcheckPath: "/"
+    ingress:
+      enabled: true
+      annotations: {}
+    service:
+      port: 80
+
+  licensifyBackedn:
```
Comment (sengi) [inline charts/licensify/values.yaml:33]:
```suggestion
  licensifyBackend:
```

File: charts/licensify/templates/ingress.yaml [hunk 2]
```diff
@@ -1,26 +1,31 @@
-{{ $app := .Values.licensifyFrontend }}
-{{ $_ := set .Values "appName" $app.name }}
+{{- range .Values.apps }}
+{{- if .ingress.enabled }}
+{{ $_ := set $.Values "appName" .name }}
```
Comment (sengi) [inline charts/licensify/templates/ingress.yaml:3]: Do you need to modify `.Values`? Might be cleaner to just define a variable, like `$appName := .name` (same as you're doing for `$svcPort`)?

**Review by sengi — APPROVED** [review]

File: charts/licensify/templates/ingress.yaml (hunk 2 above)
Comment (theseanything) [reply charts/licensify/templates/ingress.yaml:3]: Ah so the reason is to make "appName" available for the "licensify.labels", by just adding it to the Values dict we pass it. (There might be a nicer way to do that.)

File: charts/licensify/values.yaml (hunk 1 above)
Comment (theseanything) [reply charts/licensify/values.yaml:33]: Ah thank you! I couldn't workout why the diff was so weird

File: charts/licensify/templates/ingress.yaml (hunk 2 above)
Comment (sengi) [reply charts/licensify/templates/ingress.yaml:3]: Oh I see. Yeah, you probably wanna just change that partial template in _helpers.tpl to do what you want (probably just delete the `.Values` and pass it `.`) — but no biggie. As long as the labels are coming out the way you want them!

**Pull request merged**